test/sdk/restapi/http-api-test.go.nolint

```diff
@@ -105,6 +112,56 @@ func main() {
 		log.Fatalf("Could not SetAnnotation: %v\n", err)
 	}
 
+  // easy feature flag check
+	if strings.Contains(os.Getenv("FEATURE_GATES"), "PlayerTracking=true") {
```

Comment:
would a check for non-empty work just as well and be more future proof?

Reply:
or would you put a different feature's tests behind a different feature gate check since they would be promoted independently?

Reply:
> or would you put a different feature's tests behind a different feature gate check since they would be promoted independently?

Yeah, that was my thought - we might get more flags, and they would be enabled independently. Or at least, we would have the option if we decided to do more matrix style testing.